stats/stats/src/charts/lines/new_native_coins_transfers.rs

```diff
@@ -0,0 +1,105 @@
+use crate::{
+    charts::{insert::DateValue, ChartUpdater},
+    UpdateError,
+};
+use async_trait::async_trait;
+use chrono::NaiveDate;
+use entity::sea_orm_active_enums::ChartType;
+use sea_orm::{prelude::*, DbBackend, FromQueryResult, Statement};
+
+#[derive(Default, Debug)]
+pub struct NewNativeCoinTransfers {}
+
+#[async_trait]
+impl ChartUpdater for NewNativeCoinTransfers {
+    async fn get_values(
+        &self,
+        blockscout: &DatabaseConnection,
+        last_row: Option<NaiveDate>,
+    ) -> Result<Vec<DateValue>, UpdateError> {
+        let stmnt = match last_row {
+            Some(row) => Statement::from_sql_and_values(
+                DbBackend::Postgres,
+                r#"
+                SELECT 
+                    DATE(b.timestamp) as date,
+                    COUNT(*)::TEXT as value
```

Comment:
shouldn't this calculate the sum of the coin values, not the number of transactions? 
i.e. not that "5 txs with coin transfers happened", but "132 eth was transferred in a day"

Reply:
It should be count of transactions